registration/forms.py

```diff
@@ -0,0 +1,10 @@
+'''importing django forms'''
+from django import forms 
+# creating a form 
+class InputForm(forms.Form):
+    '''forms'''
+    first_name = forms.CharField(max_length = 200)
+    last_name = forms.CharField(max_length = 200)
+    email = forms.EmailField(max_length = 200)
+    password = forms.CharField(widget = forms.PasswordInput(),
```

Comment:
Also add confirm password. Password and Confirm password should be equal.

Reply:
snake case kia hy sara